<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Audio Mode</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxAudioMode">
       <item>
        <property name="text">
         <string>Microphone</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pre-Recorded</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Generated</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelFrequency">
       <property name="text">
        <string>Generated Frequency (440  Hz)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="sliderFrequency">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>20</number>
       </property>
       <property name="maximum">
        <number>4000</number>
       </property>
       <property name="singleStep">
        <number>20</number>
       </property>
       <property name="value">
        <number>440</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Codec</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBoxAudioCodec"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Sample Rate</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBoxSampleRate">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Sample Size</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="comboBoxSampleSize">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>sample Type</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="comboBoxSampleType"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Window Function</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="comboBoxWindowFunction">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Rectangular Window (None)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hanning Window</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blackman-Harris</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonQuit">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Quit</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonQuit</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
